analysis/utility_analysis.py

```diff
@@ -146,12 +146,14 @@ def _pack_per_partition_metrics(
     """
     n_metrics = len(utility_result) // n_configurations
 
+    statistics = utility_result[0]
     # Create 'result' with empty elements.
-    empty_partition_metric = lambda: metrics.PerPartitionMetrics(1, [])
+    empty_partition_metric = lambda: metrics.PerPartitionMetrics(
+        1, copy.copy(statistics), [])
```

Comment:
If we don't explicitly copy, does it pass by reference?

Reply:
yes. I've thought again it's actually fine. I've removed copying